generators/app/index.js

```diff
@@ -249,7 +258,7 @@ function logData() {
         'version': pkg.version,
         'osPlatform': os.platform(),
         'projectType': projectType,
-        'usingNodemon': addNodemon === undefined ? 'undefined' : addNodemon,
+        'usingNodemon': 'N/A',
```

Comment:
Since we removed the Nodemon option, I am going to remove this as well. Wasn't sure if it's going to affect telemetry somehow.